src/tbb/task.cpp

```diff
@@ -221,6 +221,34 @@ void notify_waiters(std::uintptr_t wait_ctx_addr) {
     governor::get_thread_data()->my_arena->get_waiting_threads_monitor().notify(is_related_wait_ctx);
 }
 
+d1::wait_tree_vertex_interface* get_thread_reference_vertex(d1::wait_tree_vertex_interface* wc) {
+    __TBB_ASSERT(wc, nullptr);
+    auto& dispatcher = *governor::get_thread_data()->my_task_dispatcher;
```

Comment:
It can be not yet initialize. Consider this example:
```cpp
tbb::task_group tg;
tg.run(); // get_thread_reference_vertex will be called but TBB not yet initialized
```